commands.pm

```diff
@@ -221,6 +221,16 @@ sub isotovideo_post {
     return isotovideo_command($c, [qw(interactive stop_waitforneedle continue_waitforneedle reload_needles)]);
 }
 
+sub get_temp_file {
+    my ($self) = @_;
+    my $path = $self->param('relpath');
+    # do not allow .. in path
+    return $self->reply->not_found if defined $path && $path =~ /^(.*\/)*\.\.(\/.*)*$/;
```

Comment:
I think you can rewrite the regex as: /\..*\./

Reply:
This won't be exactly same, but we can simplify it to simply look for /../